Gemfile

```diff
@@ -263,6 +263,8 @@ group :test do
   gem 'json-schema'
   # For using 'assigns' in tests
   gem 'rails-controller-testing'
+  # For managing the environment
```

Comment:
What do we need this for if it does not work with rspec? :nerd_face:

Reply:
*hope* for the future :grin:

Reply:
I guess hope dies last then...